src/containers/Header.tsx

```diff
@@ -64,7 +64,7 @@ export const Header = () => {
             <>
               <STooltip title='Account History' placement='bottom'>
                 <IconButton onClick={handleAccountHistory} role='link'>
-                  <Badge invisible={!transactionPending} variant='dot' color='primary' overlap='circular'>
+                  <Badge invisible={pendingTransactionCount === 0} variant='dot' color='primary' overlap='circular'>
```

Comment:
I think this is more concise, but up to u
```suggestion
                  <Badge invisible={!pendingTransactionCount} variant='dot' color='primary' overlap='circular'>
```

Reply:
dafd24225f192ca7cb45d6b30823c5035ff3afab